operator/controllers/execution/scans/scan_reconciler_test.go

```diff
@@ -0,0 +1,79 @@
+// SPDX-FileCopyrightText: the secureCodeBox authors
+//
+// SPDX-License-Identifier: Apache-2.0
+
+//go:build fast
+// +build fast
+
+package scancontrollers
+
+import (
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	executionv1 "github.com/secureCodeBox/secureCodeBox/operator/apis/execution/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+var namespace = "namespace"
+var reconciler = &ScanReconciler{}
+var _ = Describe("ScanControllers", func() {
+	Context("checkIfTTLSecondsAfterFinishedisCompleted", func() {
+		It("should return true if TTLSecondsAfterFinished is set", func() {
+			finishTime := time.Date(
+				2009, 11, 17, 20, 34, 58, 651387237, time.UTC)
+			var timeout int32 = 30
+			var scan = &executionv1.Scan{
+				ObjectMeta: metav1.ObjectMeta{
+					Namespace: namespace,
+					Name:      "nmap",
+				},
+				Spec: executionv1.ScanSpec{
+					ScanType:                "nmap",
+					Parameters:              []string{"scanme.nmap.org"},
+					TTLSecondsAfterFinished: &timeout,
+				},
+				Status: executionv1.ScanStatus{
+					FinishedAt: &metav1.Time{Time: finishTime},
+				},
+			}
+			Expect(reconciler.checkIfTTLSecondsAfterFinishedisCompleted(scan)).To(BeTrue())
+		})
```

Comment:
Just noticing this now but can we fix the spelling of this function name?

```suggestion
			Expect(reconciler.checkIfTTLSecondsAfterFinishedIsCompleted(scan)).To(BeTrue())
```